printer_zpl2/views/printing_label_zpl2.xml

```diff
@@ -0,0 +1,356 @@
+<?xml version="1.0" ?>
+<!--
+    Copyright 2016 SUBTENO-IT
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="view_printing_label_zpl2_tree" model="ir.ui.view">
+        <field name="model">printing.label.zpl2</field>
+        <field name="arch" type="xml">
+            <tree string="ZPL II Label">
+                <field name="name" />
+                <field name="model_id" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_printing_label_zpl2_form" model="ir.ui.view">
+        <field name="model">printing.label.zpl2</field>
+        <field name="arch" type="xml">
+            <form string="ZPL II Label">
+                <header>
+                    <button name="import_zpl2" string="Import ZPL2" type="object" />
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <field name="action_window_id" invisible="1" />
+                        <button
+                            name="create_action"
+                            string="Add in the 'Action' menu"
+                            type="object"
+                            attrs="{'invisible':[('action_window_id','!=',False)]}"
+                            icon="fa-plus-square"
+                            help="Display an option on related documents."
+                            class="oe_stat_button"
+                        />
+                        <button
+                            name="unlink_action"
+                            string="Remove from the 'Action' menu"
+                            type="object"
+                            attrs="{'invisible':[('action_window_id','=',False)]}"
+                            icon="fa-minus-square"
+                            help="Remove the contextual action."
+                            class="oe_stat_button"
+                        />
+                        <button
```

Comment:
replace by in `sheet`
```xml
<field name="active" invisible="1"/>
<widget name="web_ribbon" text="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
```

Reply:
Done. Thank you!

Reply:
> @AaronHForgeFlow my commit (quick move doesn't work correctly with labelary), please find the correct commit : [3e9bb51](https://github.com/OCA/report-print-send/commit/3e9bb519508436ba667c7e54705d42db6375ed19)

Thanks for the update. I will do so when your PR is merged. I have many conflicts when cherry-picking your commit in 13.0 branch, so I prefer to fix those conflicts just once.